amicleaner/core.py

```diff
@@ -203,11 +203,20 @@ def map_candidates(self, candidates_amis=None, mapping_strategy=None):
                     mapping_strategy.get("values")
                 )
                 if mapping_strategy.get("excluded"):
-                    for excluded_mapping_value in mapping_strategy.get("excluded"):
-                        if excluded_mapping_value not in mapping_value:
-                            mapping_list = candidates_map.get(mapping_value) or []
-                            mapping_list.append(ami)
-                            candidates_map[mapping_value] = mapping_list
+                    mapping_list = mapping_value.split('.')
+                    excluded_mapping_values = mapping_strategy.get("excluded")
+
+                    for value in mapping_list:
+                        is_candidate = True
```

Comment:
I think this variable should be initialised outside the loop

Reply:
Good catch 👍